solutions/quickstart-vpc/version.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = ">=1.3.0"
```

Comment:
The module was just updated to require >=1.9.0 so please also update here. It means you can use cross variable validation now too if you find an applicable use case in the DA inputs